libs/sdk-ui/src/base/react/ErrorComponent.tsx

```diff
@@ -13,6 +25,7 @@ export interface IErrorProps {
     style?: object;
     width?: any;
     height?: any;
+    clientHeight?: any;
```

Comment:
Please add a TSDoc for height and clientHeight describing the difference between them.

Reply:
I still don't quite understand this... How should the user set this value? Also is it really `any`? Should it be `number`?